src/components/modules/MainPage/LongCard/StyledLongCard.js

```diff
@@ -0,0 +1,66 @@
+import styled, { css } from "styled-components";
+
+export const Container = styled.div`
+  width:100rem;
+  height: 17.5rem;
+  height:100%;
+  margin: 0.5%;
+  font-size:1.2rem;
+  border-bottom-right-radius:40px;
+  border-bottom-left-radius:40px;
+  background-color:#C3C0C0;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23); */
+  display: flex;
+  flex-direction: column;
+  @media (orientation: portrait) {
+    width: 100%;
```

Comment:
width of 100% is always applied when using display:flex. this can be deleted

Reply:
deleted !

Reply:
![Screen Shot 2022-04-11 at 3 27 35 PM](https://user-images.githubusercontent.com/58893691/162843817-ac5bb66e-fa15-41e9-89ea-851d1469fee0.png)

Reply:
![Screen Shot 2022-04-11 at 3 35 17 PM](https://user-images.githubusercontent.com/58893691/162844394-fa95c502-0b7a-4b7d-a889-c3a4707207a9.png)